Jenkinsfile_CNP

```diff
@@ -69,15 +69,14 @@ withPipeline(type, product, component) {
 
     env.IA_CASE_DOCUMENTS_API_URL = "http://ia-case-documents-api-aat.service.core-compute-aat.internal/"
     env.IA_CASE_NOTIFICATIONS_API_URL = "http://ia-case-notifications-api-aat.service.core-compute-aat.internal/"
+    env.IA_CASE_PAYMENTS_API_URL = "http://ia-case-payments-api-aat.service.core-compute-aat.internal/"
     env.IA_IDAM_REDIRECT_URI = "https://ia-case-api-aat.service.core-compute-aat.internal/oauth2/callback"
     env.CCD_URL = "https://ia-case-api-aat.service.core-compute-aat.internal/oauth2/callback"
     env.CCD_GW_URL = "http://dm-store-aat.service.core-compute-aat.internal"
     env.DM_URL = "http://dm-store-aat.service.core-compute-aat.internal"
     env.IDAM_URL = "https://idam-api.aat.platform.hmcts.net"
     env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
 
-    installCharts()
```

Comment:
Did you mean to remove this?

Reply:
Yes - Jenkins APP  8:23 AM
We have noticed deprecated configuration in HMCTS_IAC/ia-case-api/PR-396: Build #8
installCharts() is deprecated and no longer does anything, please remove this flag before the pipeline starts failing., This configuration will stop working by 25/05/2020 00:00 AM ( in 12 days )

Reply:
ok cool.  Just saw that.  👍